flow-typed/forms.js

```diff
@@ -41,6 +42,7 @@ declare type FormElementType =
   | 'geoscapeAddress'
   | 'pointAddress'
   | 'compliance'
+  | 'civicaStreetName'
```

Comment:
IF we are going to be adding more civica integration elements in future, is there a case to be made for having one element and being able to config it to do different things?

Reply:
If we want to represent it that way we the Form Builder, no problem. However, IMO, it's very important that a `type` represents a submission structure as dealing with a form element with multiple submission structures is terrible developer UX.

Reply:
I imagine it could just be handled like 
```
...
case 'civica': {
  switch (element.civicaFieldType) {
    case 'streetName': {
      // Specific submission structure  
    }
  }
}
...
```

But I'm happy to do it as separate element types as well 👍